esrc/rescue_success_mail.php

```diff
@@ -0,0 +1,172 @@
+<?php
+// Mark all entry pages with this definition. Includes need check check if this is defined
+// and stop processing if called direct for security reasons.
+if (!defined('ESRC')) define('ESRC', TRUE);
+
+require_once '../class/db.class.php';
+require_once '../class/rescue_mail.class.php';
+require_once '../class/users.class.php';
+require_once '../class/rescue.class.php';
+require_once '../class/systems.class.php';
+require_once '../class/output.class.php';
+
+// check if the user is alliance member
+if (!Users::isAllianceUserSession())
```

Comment:
This page should check for a SAR admin/operator role too. Otherwise every corp member can call this page. We already implemented a simple role/permission system which should apply here.

Reply:
Yes, @Klensor this should be done.